### PR TITLE
DENG-2611 - Fix Looker PDTs sourced from `telemetry_derived` failing to update

### PR DESCRIPTION
**Pull request by wwyc**
https://mozilla-hub.atlassian.net/browse/DENG-2611

### CODE REVIEW

**Review by scholtzan — APPROVED** [review]

**Review by whd — COMMENTED** [review]
Reviewing this made me realize that Looker doesn't actually have the requisite access to run this (since it's per-project metadata, not per-table). That's not a policy issue (e.g. redash does have this access) so I will file a PR to grant the access needed to make this PR work, and update this ticket when that has landed.

File: generator/views/datagroups.py [hunk 1]
```diff
@@ -15,11 +15,11 @@
 
 DEFAULT_MAX_CACHE_AGE = "24 hours"
 
-# Note: INFORMATION_SCHEMA.PARTITIONS has a row with a `last_modified_time` value even for non-partitioned tables.
```
Comment (ANich) [inline generator/views/datagroups.py:18]: I suspect yes, but does this also apply for `INFORMATION_SCHEMA.TABLE_STORAGE`? We probably still want to create datagroups for non-partitioned tables.

**Review by ANich — APPROVED** [review]
LGTM, left one question

File: generator/views/datagroups.py (hunk 1 above)
Comment (ANich) [reply generator/views/datagroups.py:18]: That makes sense to me 👍 no need to re-add

**Comment by whd** [comment]
Looker should now have the relevant access.

**Pull request merged**